src/components/ProgressReport/ProgressReportInfo/ReportDetails.tsx

```diff
@@ -136,31 +136,43 @@ const ReportDetails: FC<IReportDetails> = (props) => {
 				</div>
 			)}
 
-			<div
-				className='mb-1 flex flex-col rounded-md border border-solid border-[#D2D8E0] px-4 py-2 dark:border-separatorDark dark:bg-transparent'
-				style={{
-					background: 'rgba(210, 216, 224, 0.20)'
-				}}
-			>
-				<div className='flex items-center justify-start gap-x-2'>
-					<div className='flex h-[32px] w-[32px] items-center justify-center rounded-md bg-[#F9173E]'>
-						<ImageIcon
-							src='/assets/icons/pdf-icon.svg'
-							alt='pdf.icon'
-						/>
-					</div>
-					<div className='flex flex-col gap-y-0.5'>
+			<div>
+				{report?.progress_file &&
+					(() => {
+						try {
+							const fileUrl = new URL(report.progress_file);
+							const allowedDomains = ['polkassembly.io', 'subsquare.io'];
```

Comment:
this check should be in the back-end, not FE, we're using unnecessary network bandwidth